src/ui/Assets/Styles/site.scss

```diff
@@ -50,6 +50,12 @@ $govuk-global-styles: true;
   border: 4px solid govuk-colour("red");
 }
 
+// custom table header border tickness
+.govuk-table__header {
+  border-bottom-width: 2px;
+  border-bottom-color: govuk-color("black");
```

Comment:
![image](https://user-images.githubusercontent.com/470137/44989933-697c4e00-af87-11e8-9d35-7ea32b444f46.png)

Reply:
that's not how you spell colour is it 😬